website/docs/layouts/app-frame/partials/code/how-to-use.md

```diff
@@ -0,0 +1,126 @@
+    <Frame.Footer>
+      {{! your "footer" content goes here, this is just a mock placeholder }}
+      <Doc::Placeholder @height="60px" @text="footer" @background="#fff8d2" />
+    </Frame.Footer>
+  </Hds::AppFrame>
+</div>
+```
+
+### Optional containers
+
+Depending on the UI implementation of the product where the component is used, it's possible to omit certain containers simply by not yielding them. For example, this would be the “frame” of an application that doesn't have an “header”:
+
+```handlebars
+<div class="doc-app-frame-mock-viewport">
+  <Hds::AppFrame as |Frame|>
+    <Frame.Sidebar>
+      <Doc::Placeholder @width="120px" @height="100%" @text="sidebar" @background="#e4c5f3" />
+    </Frame.Sidebar>
+    <Frame.Main>
+      <Doc::Placeholder @height="100%" @text="main" @background="#d2f4ff" />
+    </Frame.Main>
+    <Frame.Footer>
+      <Doc::Placeholder @height="60px" @text="footer" @background="#fff8d2" />
+    </Frame.Footer>
+  </Hds::AppFrame>
+</div>
+```
+
+#### Programmatic control of the containers‘ rendering
+
+Using the [exposed API of the component](#component-api), it's possible to programmatically control the rendering of some of the containers. An example of programmatic control of the rendering of the sidebar could be this:
+
+```handlebars
+<div class="doc-app-frame-mock-viewport">
+  <Hds::AppFrame as |Frame|>
+    {{! conditional control of the rendering of the "sidebar" }}
+    {{#if this.yourSidebarBooleanFlag}}
+      <Frame.Sidebar>
+        <Doc::Placeholder @width="120px" @height="100%" @text="sidebar" @background="#e4c5f3" />
+      </Frame.Sidebar>
+    {{/if}}
+    <Frame.Main>
+      <Doc::Placeholder @height="100%" @text="main" @background="#d2f4ff" />
+    </Frame.Main>
+    <Frame.Footer>
+      <Doc::Placeholder @height="60px" @text="footer" @background="#fff8d2" />
+    </Frame.Footer>
+  </Hds::AppFrame>
+</div>
+```
+
+If for some reason it's not possible to use conditional logic to control the yielding of the containers, we provide an alternative way using special `has[Container]` arguments to programmatically control the rendering (see the [component API](#component-api) specifications for details):
+
+```handlebars
+<div class="doc-app-frame-mock-viewport">
+  <Hds::AppFrame @hasSidebar={{false}} as |Frame|>
+    <Frame.Sidebar>
+      <Doc::Placeholder @width="120px" @height="100%" @text="sidebar" @background="#e4c5f3" />
+    </Frame.Sidebar>
+    <Frame.Main>
+      <Doc::Placeholder @height="100%" @text="main" @background="#d2f4ff" />
+    </Frame.Main>
+    <Frame.Footer>
+      <Doc::Placeholder @height="60px" @text="footer" @background="#fff8d2" />
+    </Frame.Footer>
+  </Hds::AppFrame>
+</div>
+```
+
+### Modals container
+
+We provide also an extra container that can be used to display content that sits on top of all the others elements of the page (typically modal elements):
+
+!!! Info
+
+If the “modal“ container is empty, a `display: none` style is applied to it.
+!!!
```

Comment:
Should this be a warning banner instead of an informational one?

Reply:
No, there's nothing "dangerous". It's just an information about the fact that people should not worry that it may interact with the user clicks when it's empty.